packages/synapse-react-client/src/components/FullWidthAlert/FullWidthAlert.tsx

```diff
@@ -96,14 +88,12 @@ function FullWidthAlert(props: FullWidthAlertProps) {
     description,
     primaryButtonConfig,
     secondaryButtonConfig,
-    show,
+    show = true,
```

Comment:
Maintain the same default value for `show` as in [react-bootstrap](https://react-bootstrap.github.io/components/alerts/#alert-props). Ensures that components that conditionally display the FullWidthAlert rather than toggling the `show` prop will still work as expected, e.g. TwoFactorEnrollmentForm